src/pages/editlist/EditListPage.tsx

```diff
@@ -184,18 +214,22 @@ export const EditListPage:FC = () => {
           </Pane>
         </Paneset>
       </Layer>
-      <ConfirmationModal
-        confirmLabel={t('list.modal.keep-edit')}
-        cancelLabel={t('list.modal.cancel-edit')}
-        heading={t('list.model.sure-heading')}
-        message={t('list.modal.confirm-cancel-message')}
+      <CancelEditModal
         onCancel={() => {
           setShowConfirmCancelEditModal(false);
           backToList();
         }}
-        onConfirm={() => setShowConfirmCancelEditModal(false)}
+        onKeepEdit={() => setShowConfirmCancelEditModal(false)}
         open={showConfirmCancelEditModal}
       />
+      <ConfirmDeleteModal
+        listName={listName}
+        onCancel={() => setShowConfirmDeleteModal(false)}
+        onConfirm={() => {
+          deleteListHandler();
+        }}
```

Comment:
The tiniest of things 😄: the formatting here is inconsistent with the surrounding code